An Untitled Mining Game/An Untitled Mining Game/objects/obj_structure_parent/Draw_0.gml

```diff
@@ -0,0 +1,11 @@
+draw_sprite(draw_spr, 0, x + 0, y + 0);
+
+draw_set_font(fnt_smalldigits);
+draw_set_colour($FFFFFFFF & $ffffff);
+var l1AF7C74B_0=($FFFFFFFF >> 24);
+draw_set_alpha(l1AF7C74B_0 / $ff);
+
+draw_text(x + 8, y + 8,  + string(structure_level));
+
+//Timed delay and thermo
+scr_show_and_craft(id);
```

Comment:
I'm very curious as to what the rightward bitshift (ln 5) here is up to! 0xFFFFFFFF is the ceiling for 32-bit integers (thus, all bits are 1), so shifting to the right 24 bits would also result in 0x000000FF, which is 255. draw_set_alpha would always receive 1 as its parameter if this is true. (255/255)

Reply:
That's an artifact of when I converted my project from Drag'n'Drog to pure GML, and mt thoughts are it doesn't break anything so I'm going to just leave it alone since its just modifying a local var its selfcontained.